__tests__/app.test.js

```diff
@@ -86,3 +87,33 @@ describe("GET", () => {
       });
   });
 });
+
```

Comment:
can we add a 200 test for  valid ID, but has no comments responds with an empty array of comments